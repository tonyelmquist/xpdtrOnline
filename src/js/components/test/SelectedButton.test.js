import React from "react";
import renderer from 'react-test-renderer'
import SelectedButton from '../SelectedButton'

const currency = {"baseCurrency": "USD", "quoteCurrency": "NOK", "rate": "7.75", "selected": false, "timePeriod": "2017"};

it('should render correctly', () => {
  const component = renderer.create(<SelectedButton currency={currency}/>)
  expect(component.toJSON()).toMatchSnapshot()
})