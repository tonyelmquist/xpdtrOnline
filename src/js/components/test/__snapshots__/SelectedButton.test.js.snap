// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<button
  className="ui grey button currency-button"
  disabled={undefined}
  onClick={[Function]}
  role="button"
  tabIndex={undefined}
>
  <span
    className="currency-name"
  >
    USD
  </span>
  <span
    className="flag-span"
  >
    <i
      className="as flag"
    />
  </span>
  <span
    className="currency-rate"
  >
    current rate in NOK: 
    7.75
  </span>
  <span
    className="currency-star"
  >
    <i
      aria-hidden="true"
      className="star small icon unselected"
    />
  </span>
</button>
`;
